projects/ngx-fixture/src/matchers-types.d.ts

```diff
@@ -0,0 +1,7 @@
+declare namespace jasmine {
+  interface Matchers<T> {
+    toHaveClass(className: string): boolean;
```

Comment:
In the jasmine .d.ts file, some matchers have an additional `expectationFailOutput?: any` argument, but some others don't. I don't know if we should add one or not.